iree/compiler/Conversion/LinalgToSPIRV/KernelDispatchUtils.cpp

```diff
@@ -412,6 +412,42 @@ LogicalResult getOpLaunchConfig(linalg::ConvOp op,
   return success();
 }
 
+template <>
```

Comment:
Just use a single macro for this?

Reply:
It needs something like `std::enable_if`. I tried it once, but couldn't get it work. I will give it another shot.

Reply:
Oh I see your point, use macro, not SFINAE

Reply:
I think for Mali we only support 2-D NHWC for now. Let's don't add other cases as the configurations are expected to be different.

Reply:
We actually needs 1/2/3-D support because some tensorflow tests cover them. I will add some tests for it.

Reply:
done